### ISSUE TITLE
While creating virtual environment VS Code does not prompt with venv notification

### ISSUE THREAD

**Issue by Debkmr** [microsoft/vscode-docs#4101]
While creating virtual environment VS Code does not prompt with venv notification
![Capture](https://user-images.githubusercontent.com/73872111/97955601-8e211a00-1dcc-11eb-99ad-7ad2c538cb89.JPG)

**Comment by JoshuaPartlow** [comment]
Hi @Debkmr - Just to confirm, you're not seeing a VS Code notification that a virtual environment has been created? The one that should offer to let you set it as the default for the workspace?

**Comment by Debkmr** [comment]
Yes, you got it right there, can you tell me how could I fix that. I am using python 3.9 version.

**Comment by Aryankpoor** [comment]
I faced this issue earlier when vscode was running too many processes at the same time, so the notification comes a little late

**Comment by Debkmr** [comment]
I am not sure, I am still looking for a work around.

**Comment by Aryankpoor** [comment]
Is your vscode the latest version?

**Comment by Debkmr** [comment]
Yes, latest VS code and python 3.9

**Comment by JoshuaPartlow** [comment]
One more question, are you able to activate the environment from the VS Code prompt? I notice in your image that it failed, but it also looks like the environment name wasn't right in that attempt venv vs .venv. Thanks!

**Comment by Debkmr** [comment]
I tried with .venv and venv as well.

**Comment by JoshuaPartlow** [comment]
Small thing to check, but when you run activate, are you definitely running activate.ps1 (vs activate.bat). Can you try running with the full extension `.venv\scripts\activate.ps1`

**Comment by Debkmr** [comment]
![Error](https://user-images.githubusercontent.com/73872111/98322554-b2b80480-200d-11eb-8525-848fcede733d.JPG)
This is the error I am getting

**Comment by JoshuaPartlow** [comment]
Hi @Debkmr - That's an expected issue when running a PowerShell script as execution is restricted by default. You can read about the issue and some solutions [here](https://github.com/Microsoft/vscode-python/issues/2559).  Essentially, I believe you can either change the execution policy for the process (as discussed in the VS Code Python tutorial [here](https://code.visualstudio.com/docs/python/python-tutorial#_install-and-use-packages)) or there's also a Windows setting you can change to allow the running of local unsigned PowerShell scripts (start -> search developer settings -> PowerShell) in general. Alternatively, you could use CMD as your integrated terminal with the .bat file instead of PowerShell. Does that help?

**Comment by JoshuaPartlow** [comment]
If this is still an ongoing issue, please reopen.

**Comment by standi98** [comment]
I have this exact issue at the moment. I can activate the viritual enviroment once it is created, but in order to set it as the default one for my workpace I have to manualy add it to the "settings.json" file. It takes a cuple of minutes from the moment it is created to when it shows up in the list over interpreters, and the prompt for adding it to the workspace never shows up.

**Comment by gregvanl** [comment]
@luabud for visibility
